tvshows/src/components/features/review/ReviewList/ReviewList.tsx

```diff
@@ -11,7 +11,7 @@ export const ReviewList = ({ reviewList }: IReviewListProps) => {
   return (
     <Fragment>
       <Box textAlign="left" mt={10} mb={5}>
-        <Heading fontSize='3xl'>Reviews</Heading>
+        <Text textStyle="h2">Reviews</Text>
```

Comment:
I see you replacing Headings with Text components. If you semantically need a Heading component, it is OK to use that, doesn't need to be Text. Of course, if semantically you want a Text component, use it.